### ISSUE TITLE
The date displaying on Workflow Run Schedule is incorrect

### ISSUE THREAD

**Issue by mbjones** [NCEAS/metacat#742]
---

Author Name: **Jing Tao** (Jing Tao)
Original Redmine Issue: 4866, https://projects.ecoinformatics.org/ecoinfo/issues/4866
Original Date: 2010-03-03
Original Assignee: Jing Tao

---

I tried to schedule a workflow run on the scheduler interface. The start time is 03/03/2010 15:30:00 and end time is 03/03/2010 17:00:00. After clicking Schedule button, the result shows:

Start Time                         End Time
30/03/10 15:30:00 PST              00/03/10 17:00:00 PST

So the date displaying is incorrect. However, on 03/03/2010 15:30:00 the workflow did a run. So it is just displaying problem.

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Jing Tao** (Jing Tao)
Original Date: 2010-03-05T00:56:44Z

---

This is a tricky bug.

The problem is on the DateUtil class of the utilities module. When scheduler transforms a Timestamp object to a string which will be displayed, it calls getHumanReadable method on that class. This method will format the Timestamp object. The format was "mm/dd/yy HH:mm:ss z". However, lower case "mm" means minutes, upper case "MM" means month. So the display was not correctly. When I changed the format to be  "MM/dd/yy HH:mm:ss z". It works.

I fixed a bug on svn trunk of the utilites module. But I haven't tagged it yet. I will open another bug to discuss which tag or version of the utilities module should be used on 1.9.2 release.
